tests/integ/modin/test_modin_stored_procedures.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import modin.pandas as pd  # noqa: F401
```

Comment:
Why is noqa: F401 needed here?  It looks like pd is used.

Reply:
True it's not needed, removed